mpyl_config.yml

```diff
@@ -14,9 +14,9 @@ slack:
     success: 'thug-parrot'
     failure: 'sadparrot'
 jira:
-  site: !ENV ${JIRA_URL}
-  userName: !ENV ${JIRA_USER_PASSWORD_USR:jira_user}
-  password: !ENV ${JIRA_USER_PASSWORD_PSW:jira_password}
+  site: 'https://vandebron.atlassian.net'
```

Comment:
the env var should also be removed again from the jenkins yaml in the platform repo then

Reply:
https://github.com/Vandebron/mpyl/pull/84/commits/a9327aed0c62dee25e76a07711ea2f48d92898b3